docs/mesos/UsageGuideDockerContainerizer.md

```diff
@@ -8,165 +8,105 @@
-
+Run the following command on any agent to create a Docker network with Calico:
 ```
-docker network create --driver calico --ipam-driver calico databases 
+docker network create --driver=calico --ipam-driver=calico my-calico-net
 ```
 
-#### View the policy associated with the network
+#### View Network Policy
 
 You can use the `calicoctl profile <profile> rule show` to display the
-rules in the profile associated with the `databases` network.
+rules in the profile associated with the `my-calico-net` network.
 
 The network name can be supplied as the profile name and the `calicoctl` tool
 will look up the profile associated with that network.
 
 ```
-$ ETCD_AUTHORITY=<IP address>:4001
-$ calicoctl profile databases rule show
+$ export ETCD_AUTHORITY=$ETCD_LISTEN_URL
+$ calicoctl profile my-calico-net rule show
 Inbound rules:
-   1 allow from tag databases
+   1 allow from tag my-calico-net
 Outbound rules:
    1 allow
 ```
 
-As you can see, the default rules allow all outbound traffic and accept inbound
-traffic only from containers attached the "databases" network.
+The default rules allow all outbound traffic and accept inbound
+traffic only from containers attached the "my-calico-net" network.
 
 > Note that when managing profiles created by the Calico network driver, the
 > profile tag and network name can be regarded as the same thing.
 
-For more information no how to configure your Calico profiles, check out
-the section on [Configuring the network policy in our Advanced Policy guide]
-(../calico-with-docker/docker-network-plugin/AdvancedPolicy.md#configuring-the-network-policy).
+For more information no how to configure your Calico profiles, see [Configuring Advanced Network Policy Guide](../calico-with-docker/docker-network-plugin/AdvancedPolicy.md#configuring-the-network-policy).
 
 ## Launching Containers
-
-With your networks configured, it is trivial to launch a Docker container 
-through Mesos using the standard Marathon UI and API.
-
-#### Launching a container through the UI
-
-You can launch Docker task through the Marathon UI at `MARATHON_IP:8080`.
-Select an arbitrary(*) network (Bridge or Host), and then provide the
-following additional parameter (under the Docker options):
-
-```
-Key = net
-Value = <network name>
-```
-
-Where `<network name>` is the name of the network, for example "databases".
-
-> (*) The selection is arbitrary because the additional net parameter overrides
-> the selected network type.
-
-#### Launching a container with a JSON blob
-
-To launch a Calico-networked application using the Marathon API
-with a JSON blob, simply include the net parameter in the request.
-For example:
-
+With your networks configured, it is trivial to launch a calico-networked Docker container using the standard Marathon API. In your Marathon application definition, simply set `container.docker.network` to `USER`, and specify which network the task should join in `ipAddress.networkName`:
```

Comment:
It's a personal thing - but I'm never a fan of saying "simply" do this ... i'd just remove the simply.